api/v1alpha08/sonataflowplatform_types.go

```diff
@@ -121,6 +148,8 @@ type SonataFlowPlatformTriggerRef struct {
 	// Namespace of the Trigger
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Trigger_NS"
 	Namespace string `json:"namespace"`
+	//+operator-sdk:csv:customresourcedefinitions:type=status,displayName="dbMigrationStatus"
+	SonataFlowPlatformDBMigrationStatus *SonataFlowPlatformDBMigrationStatus `json:"sonataFlowPlatformDBMigrationStatus,omitempty"`
```

Comment:
Why are you adding this field here?

Reply:
I think this was added by mistake and is removed now.